src/http.c

```diff
@@ -106,22 +109,40 @@ void http_update(struct http *http,
                  const void *data,
                  unsigned int data_len,
                  unsigned int report_http) {
+
+    struct http_message *message = NULL;
+    char raw_header[HTTP_MAX_LEN] = {0};
```

Comment:
raw_header creates a big stack. Suggest calloc the memory

Reply:
Agree